packages/xrpl/HISTORY.md

```diff
@@ -3,11 +3,13 @@
 Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xrpl-announce) for release announcements. We recommend that xrpl.js (ripple-lib) users stay up-to-date with the latest stable release.
 ## Unreleased
 
+### Fixed
+* Fixed `ServerState.transitions` typing, it is now a string instead of a number. (Only used in return from `server_state` request)
+* Added `destination_amount` to `PathOption` which is returned as part of a `path_find` request
+
 ### Added
 * Null and undefined values in transactions are now treated as though the field was not passed in.
 
-### Fixed
```

Comment:
Why was this removed?

Reply:
I just accidentally swapped the order of Fixed and Added during the merge (in practice all the bullets were still there)

Flipping it back though because I think generally Added should come before Fixed